action/cache.go

```diff
@@ -21,3 +23,17 @@ func CacheClear() {
 
 	msg.Info("Glide cache has been cleared.")
 }
+
+// CachePath returns the path to a cached dependency's sources.
+func CachePath(importPath string) {
```

Comment:
I amended the commit to reflect the proper comment. It was written before I realized your functions prefer to print/die instead of return.